src/layouts/ReferenceItemLayout.astro

```diff
@@ -169,6 +171,23 @@ const seenParams: Record<string, true> = {};
           </>
         )
       }
+      {
+        (entry.data.return) && (
+          <>
+            <div class="mb-xl">
+              <h2 class="mb-md text-h3">{t("Return")}</h2>
```

Comment:
Looks like on the old site it was "Returns" with an s, maybe we can do the same here too?

<img width="905" alt="image" src="https://github.com/user-attachments/assets/170cb3f6-45cd-4600-b1b9-ab3b452b11cc">


```suggestion
              <h2 class="mb-md text-h3">{t("Returns")}</h2>
```

Reply:
@davepagurek 
Thanks for reply.

This is a suggestion for improvement.

Indeed, the old version was with "s".
I assume this “s” means plural, but can the returned value be plural?

- It is certainly possible to have multiple meanings by using arrays or Json, but I think that should be explained in the Description.

- As a property of the .mdx file is “return:”.

- Also, for example, there is no “s” in processing.org.
  (I haven't checked all the pages... For example, https://processing.org/reference/cos_.html .)

Please consider the above!

If you need the “s”, I will modify the necessary part :-)

Reply:
Returns with an s is I think more common, e.g. [Tone.js](https://tonejs.github.io/docs/15.0.4/classes/Context.html#createWaveShaper), [React](https://react.dev/reference/react/useCallback#returns), [matter.js](https://brm.io/matter-js/docs/classes/Constraint.html), and [lodash](https://lodash.com/docs/4.17.15#now). I think the reason for it isn't to consider "return" as a plural noun, but rather as a verb, e.g. "this function returns a number", or just "returns: number" for short in this case. In this case the s just means present tense, "it returns" as opposed to "it returned" in the past.

Reply:
@davepagurek 
I have updated the branch.
Please re-review.
Thank you.